src/rez/cli/selftest.py

```diff
@@ -46,7 +53,7 @@ def setup_parser(parser, completions=False):
     # make an Action that will append the appropriate test to the "--test" arg
     class AddTestModuleAction(argparse.Action):
         def __call__(self, parser, namespace, values, option_string=None):
-            name = option_string.lstrip('-')
+            name = option_string.lstrip("-")
```

Comment:
I'm noticing a bunch of string conversions. Is that intentional or you have your editor set to `black` things automatically? Not that I'm against black, but it does clutter up the PR, I'd rather have a formatting-specific PR if one is needed.